src/kubernetes_cleanup_pods_by_name/kubernetes_cleanup_pods_by_name/config.py

```diff
@@ -0,0 +1,79 @@
+import os
+import argparse
+
+# add /src to sys.path for importing modules in 'lib' dir
+# an alternative to having to add context.py to every module
+# https://stackoverflow.com/a/4383597
+import sys
+sys.path.insert(1, os.path.abspath('src'))
+
+from lib.logging import *
```

Comment:
I decided to import `logging` through `lib.logging` and remove the extra import.

Reply:
wonder how you feel about [import *](https://stackoverflow.com/questions/2386714/why-is-import-bad)

Reply:
Dont feel that good about it after reading that comment and I certainly wouldn't want to make someone hate their job. Will remove and `*` and revert back to the direct import.